scripts/apps/dashboard/world-clock/configuration.html

```diff
@@ -19,7 +19,7 @@
                 <div class="grid__item grid__item--col-12">
                     <ul class="list-group">
                         <li class="list-group-item" ng-repeat="zone in configuration.zones">
-                            <a href="">{{ zone.replace('_', ' ')}}</a>
+                            <a href="">{{ zoneLabels[zone].replace('_', ' ')}}</a>
```

Comment:
Why are you replacing the underscore? It shouldn't be in the translations strings in the first place, should it?